gitcommands.go

```diff
@@ -423,6 +423,54 @@ func gitSquashPreviousTwoCommits(message string) (string, error) {
 	return runDirectCommand("git reset --soft HEAD^ && git commit --amend -m \"" + message + "\"")
 }
 
+func gitRebaseTwoCommits(branchName string, shaValue string, action string) (string, error) {
```

Comment:
Seems like you're not sending in an action argument anymore, so this argument can just take the shaValue. Also, I'm not quite sure what this function is doing by reading its name. Could we use a more expressive name? I don't mind longer function names as long as they explaining what they do. If this function only squashes a commit into the above commit, perhaps gitSquashCommitUp?

I'm not completely familiar with rebasing though, so I'm not exactly sure what this function does.